README.md

```diff
@@ -482,8 +482,8 @@ Commands:
 
 To run Cromwell in a notebook instance:
```

Comment:
Above, can we change the "generate-config" description to something like:

> generate-config  Generate a Cromwell configuration file (cromwell.conf) for use in a Terra workspace.


I'm wondering if we should be even more precise "for use on a Terra workspace cloud environment."

You can't use the generated config file anywhere else, right? The config file points to the Pet Service Account, and AFAIK those credentials are only available on a workspace cloud environment, so I think it is worth being explicit about that.

Reply:
Yes, you're right. The config specifically includes the pet SA and the GCP project of the workspace. I went ahead and made your suggested changes.